src/commands/service/create-revision.ts

```diff
@@ -69,12 +71,10 @@ export const getDefaultRings = (
         .filter((ring) => !!ring.isDefault)
         .map((ring) => ring.branch);
   if (defaultRings.length === 0) {
-    throw Error(
-      `Default branches/rings must either be specified in ${join(
-        __dirname,
-        "bedrock.yaml"
-      )} or provided via --target-branch`
-    );
+    throw buildError(errorStatusCode.VALIDATION_ERR, {
+      errorKey: "service-create-revision-cmd-err-default-branch-missing",
+      values: [join(__dirname, "bedrock.yaml")],
```

Comment:
i believe `join` should be `path.join`

Reply:
@dennisseah it's imported above from the module:

line 2:
```import { join } from "path";```

Reply:
got it thanks